Makefile

```diff
@@ -57,7 +57,7 @@ docker:
 # This target works only on Unix-like terminals.
 proto:
 	rm -rf www/grpc/gen
-	sed -i 's/{{ VERSION }}/$(VERSION)/g' www/grpc/buf/openapi.config.yaml
+	sed -i '' 's/{{ VERSION }}/$(VERSION)/g' www/grpc/buf/openapi.config.yaml
```

Comment:
Please remove this line. Probably it is from old code.

Reply:
"[www/grpc/buf/openapi.config.yaml](https://github.com/pactus-project/pactus/blob/main/www/grpc/buf/openapi.config.yaml)" has no template to get replaced. Check the main branch